### DISCUSSION TITLE
centos8里面为什么没有fcitx5

### DISCUSSION THREAD

**Discussion by youmoula** [fcitx/fcitx5#318] [General]
用了 
`yum search fcitx5` and `dnf search fcitx5` 没有任何内容

**Comment by karuboniru** [comment]
因为没人打包 -_-
我有空看看，但是我猜 epel8 里面缺的依赖应该不少，可能要不少时间跟别的 epel 维护者沟通

**Reply by terwer** [reply]
同求，输入法是刚需啊，没有输入法系统还怎么玩，我安装的最新rocky8.6，rhel的下游，也还是没有fcitx5，悲伤(⊙︿⊙)

**Comment by wengxt** [comment]
这个情况，你可以选择安装 flatpak 版的 fcitx5，装上 fcitx4 的 im module （fcitx-gtk2,3,qt5)，设置一下环境变量和启动应该就可以用了。

**Reply by wengxt** [reply]
https://fcitx-im.org/wiki/Install_Fcitx_5#Install_Fcitx_5_from_Flatpak

**Reply by terwer** [reply]
感谢，能用了。大部分应用都完美了，只是jetbrains系列不行，有issue
https://youtrack.jetbrains.com/issue/IDEA-290661/fcitx5-is-no-longer-activated-in-idea-other-jetbrains-IDEs-don-t

**Reply by wengxt** [reply]
jetbrains 多半是你 xim 环境变量配置问题，应该是能用的。
输入窗口弹出位置的话那是另一个问题

**Reply by rockyicer** [reply]
> 感谢，能用了。大部分应用都完美了，只是jetbrains系列不行，有issue https://youtrack.jetbrains.com/issue/IDEA-290661/fcitx5-is-no-longer-activated-in-idea-other-jetbrains-IDEs-don-t

请教 稍微详细一点的操作流程
